lib/clearsale/order.rb

```diff
@@ -1,122 +1,16 @@
-        build_payment_data(b, order, payment, user)
-      end
-
-      builder.tag!('Items') do |b|
-        order.order_items.each do |order_item|
-          build_item(b, order_item)
-        end
-      end
-    end
-
-    def self.build_user_data(builder, user, billing_address)
-      builder.tag!('ID', user.id)
-      builder.tag!('Type', 1) # Pessoa Física
-      builder.tag!('LegalDocument1', user.cpf.gsub(/[\.\-]*/, '').strip)
-      builder.tag!('Name', user.full_name)
-      builder.tag!('BirthDate', user.birthdate.to_time.strftime("%Y-%m-%dT%H:%M:%S")) if user.birthdate.present?
-      builder.tag!('Email', user.email)
-      builder.tag!('Genre', user.gender.downcase)
-      build_address(builder, billing_address)
-      builder.tag!('Phones') do |b|
-        build_phone(b, user)
-      end
-    end
-
-    def self.build_address(builder, address)
-      builder.tag!('Address') do |b|
-        builder.tag!('Street', address.street_name)
-        builder.tag!('Number', address.number)
-        builder.tag!('Comp', address.complement)
-        builder.tag!('County', address.neighborhood)
-        builder.tag!('City', address.city)
-        builder.tag!('State', address.state)
-        builder.tag!('ZipCode', address.postal_code)
-      end
-    end
-
-    def self.build_phone(builder, user)
-      if user.phone.present?
-        stripped_phone = user.phone.gsub(/\(*\)*\s*\-*/, '')
-
-        builder.tag!('Phone') do |b|
-          b.tag!('Type', 0) # Undefined
-          b.tag!('DDD', stripped_phone[0..1])
-          b.tag!('Number', stripped_phone[2..-1])
-        end
-      end
-    end
-
-    def self.build_payment_data(builder, order, payment, user)
-      builder.tag!('Payment') do |b|
-        paid_at = order.paid_at || Time.current
-
-        b.tag!('Date', paid_at)
-        b.tag!('Amount', payment.amount)
-
-        #is_credit_card
-        b.tag!('PaymentTypeID', 1)
-
-        b.tag!('QtyInstallments', order.installments)
-
-        b.tag!('CardNumber', payment.card_number)
-        b.tag!('CardBin', payment.card_number[0..5])
-        b.tag!('CardType', CARD_TYPE_MAP.fetch(payment.acquirer.to_sym, 4)) # Failover is 'outros'
-        b.tag!('CardExpirationDate', payment.card_expiration)
-        b.tag!('Name', payment.customer_name)
-        b.tag!('LegalDocument', user.cpf.gsub(/[\.\-]*/, '').strip)
-
-        build_address(b, order.billing_address)
-      end
-    end
 
-    def self.build_item(builder, order_item)
-      builder.tag!('Item') do |b|
-        b.tag!('ID', order_item.product.id)
-        b.tag!('Name', order_item.product.name)
-        b.tag!('ItemValue', order_item.price)
-        b.tag!('Qty', order_item.quantity)
-        b.tag!('CategoryID', order_item.product.category.id) if order_item.product.category.try(:id).present?
-        b.tag!('CategoryName', order_item.product.category.name) if order_item.product.category.try(:name).present?
-      end
+    def self.build_orders_request_body(orders)
```

Comment:
This should be an instance method and you should pass in the order(s) at initialization.
```ruby
order = Clearsale::Order.new(order)
order.request_body
```
Also as noted by the example, this methods name isn't very semantic. Following the pattern above, `request_body` is preferred.